tests/acceptance/features/apiCollaboration/checkFileInfo.feature

```diff
@@ -1240,3 +1240,618 @@ Feature: check file info with different wopi apps
         }
       }
       """
+
+
+  Scenario: check the information of a project space file with fakeOffice
```

Comment:
imo file in project space sounds better than project space file. 
```suggestion
  Scenario: check info of a file in  project space  with fakeOffice
```

Reply:
whichever you prefer. both seems ok to me